[CI] CoreWithMultipleProjectsClientYamlTestSuiteIT test {yaml=nodes.stats/11_indices_metrics/Metric - blank for indices mappings} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #9227 / windows-2022_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/jwik7machx7iq)
- [elasticsearch-pull-request #79383 / part-2](https://gradle-enterprise.elastic.co/s/sk7ovyajofwn6)
- [elasticsearch-pull-request #78508 / part-2](https://gradle-enterprise.elastic.co/s/a4wmgap7illcm)
- [elasticsearch-pull-request #78454 / part-2](https://gradle-enterprise.elastic.co/s/wl2pygrgmgy42)

**Reproduction Line:**
```
gradlew ":x-pack:qa:multi-project:core-rest-tests-with-multiple-projects:yamlRestTest" --tests "org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT" -Dtests.method="test {yaml=nodes.stats/11_indices_metrics/Metric - blank for indices mappings}" -Dtests.seed=B4EE436D22E9574D -Dtests.locale=shi-Latn -Dtests.timezone=Etc/GMT -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dnodes.stats%2F11_indices_metrics%2FMetric%20-%20blank%20for%20indices%20mappings%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [nodes.stats/11_indices_metrics:412]: 
Expected: <0>
     but: was <33>
```

**Issue Reasons:**
- [main] 4 failures in test test {yaml=nodes.stats/11_indices_metrics/Metric - blank for indices mappings} (0.4% fail rate in 995 executions)
- [main] 3 failures in step part-2 (0.8% fail rate in 385 executions)
- [main] 3 failures in pipeline elasticsearch-pull-request (0.8% fail rate in 387 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -567,12 +567,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeIT
   method: test
   issue: https://github.com/elastic/elasticsearch/issues/130067
-- class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
-  method: test {yaml=nodes.stats/11_indices_metrics/Metric - blank for indices mappings}
-  issue: https://github.com/elastic/elasticsearch/issues/130772
-- class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
-  method: test {yaml=search.vectors/70_dense_vector_telemetry/Field mapping stats with field details}
-  issue: https://github.com/elastic/elasticsearch/issues/130777
 
 # Examples:
 #

--- a/x-pack/qa/multi-project/core-rest-tests-with-multiple-projects/src/yamlRestTest/java/org/elasticsearch/multiproject/test/CoreWithMultipleProjectsClientYamlTestSuiteIT.java
+++ b/x-pack/qa/multi-project/core-rest-tests-with-multiple-projects/src/yamlRestTest/java/org/elasticsearch/multiproject/test/CoreWithMultipleProjectsClientYamlTestSuiteIT.java
@@ -44,6 +44,8 @@ public class CoreWithMultipleProjectsClientYamlTestSuiteIT extends MultipleProje
         .setting("xpack.ml.enabled", "false")
         .setting("xpack.license.self_generated.type", "trial")
         .setting("xpack.security.autoconfiguration.enabled", "false")
+        // disable ILM history, since it disturbs tests using _all
+        .setting("indices.lifecycle.history_index_enabled", "false")
         .user(USER, PASS)
         .systemProperty("es.queryable_built_in_roles_enabled", "false")
         .feature(FeatureFlag.TIME_SERIES_MODE)

